recipes/starshot_editoria11y/recipe.yml

```diff
@@ -0,0 +1,10 @@
+name: Editoria11y support
+type: Starshot
+description: Installs and configures editoria11y module.
+install:
+  - editoria11y
```

Comment:
If `editoria11y` exposes any permissions, we probably want to update the main `starshot` recipe to ensure those permissions are added to the relevant roles (probably any role that can create or edit content should be able to use it).

Reply:
It looks like the default config for editoria11y permissions is to grant "View Editoria11y checker" and "Mark as hidden in Editoria11y"  to the authenticated user, which then includes the Content editor role. This could be okay for the site at the moment unless there is a plan to allow non-editors to create accounts.
The other permissions exposed are only granted to the administrator role.
Will recipes allow us to remove permission from a role? If so, I could remove the permissions from the Authenticated role and add it to Content editor.

Reply:
> It looks like the default config for editoria11y permissions is to grant "View Editoria11y checker" and "Mark as hidden in Editoria11y" to the authenticated user

If that's the case, then that's probably okay for now, but can you link me to the spot in Editoria11y's code where it makes this decision?

Reply:
From what I can see, it looks like it is set here:
https://git.drupalcode.org/project/editoria11y/-/blob/2.1.x/editoria11y.install?ref_type=heads#L20